gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/OrganizationRepository.java

```diff
@@ -26,4 +27,5 @@
 public interface OrganizationRepository extends CrudRepository<Organization, String> {
     Long count() throws TechnicalException;
     Set<Organization> findByHrids(Set<String> hrids) throws TechnicalException;
+    Optional<Organization> findByCockpit(String cockpitId) throws TechnicalException;
```

Comment:
in the service and mongo implem we have `findByCockpitId`, should we keep the same naming ?

Reply:
Ok, but I will refactor EnvironmentService also ? (Its the same naming)